modules/capi-yaml-infrastructure-rhosp.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster_api_provider_configurations/cluster-api-config-options-rhosp.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="capi-yaml-infrastructure-rhosp_{context}"]
+= Sample YAML for a Cluster API infrastructure cluster resource on {rh-openstack}
+
+The infrastructure cluster resource is provider-specific and defines properties that are shared by all the compute machine sets in the cluster, such as the region and subnets.
+The compute machine set references this resource when creating machines.
+
+[IMPORTANT]
+====
+Only the parameters in the following example are validated to be compatible with {product-title}.
+Other parameters, such as those documented in the upstream link:https://cluster-api-openstack.sigs.k8s.io/clusteropenstack/configuration#optional-configuration[{cap-openstack-first}] book, might cause undesired behavior.
+====
+
+[source,yaml]
+----
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: OpenStackCluster # <1>
+metadata:
+  name: <cluster_name> # <2>
+  namespace: openshift-cluster-api
+  labels:
+    cluster.x-k8s.io/cluster-name: <cluster_name>
+spec:
+  controlPlaneEndpoint: <control_plane_endpoint_address> # <3>
+  disableAPIServerFloatingIP: true
+  tags:
+    - openshiftClusterID=<cluster_name>
+  network:
+    id: <api_service_network_id> # <4>
+  externalNetwork:
+    id: <floating_network_id> # <5>
+  identityRef:
+    cloudName: openstack
+    name: openstack-cloud-credentials
+----
+<1> Specify the infrastructure kind for the cluster.
+This value must match the value for your platform.
+<2> Specify the cluster ID as the name of the cluster.
+<3> Specify the IP address of the control plane endpoint and the port used to access it.
+<4> Specify the UUID of the default network to use for machines that do not specify ports.
++
+[NOTE]
+====
+This feature might be removed in a future release.
+To prevent issues due to the removal of this feature, specify at least one port in the machine specification instead of relying on this feature.
```

Comment:
This sentence needs something; not sure what. The phrase _instead of relying on this feature_ seems rather negative: Don't rely on this!! 
```suggestion
To prevent issues due to the removal of this feature, specify at least one port in the machine specification instead of solely relying on this feature.
```